### PR TITLE
fix(oui-tile): normal word break for oui-paragraph

### PR DESCRIPTION
**Pull request by JeremyDec**

### CODE REVIEW

File: packages/oui-tile/tile.less [hunk 1]
```diff
@@ -89,6 +89,10 @@
     margin-left: 0;
     word-break: break-all;
 
+    & .oui-paragraph {
```
Comment (AxelPeter) [inline packages/oui-tile/tile.less]: This should be set directly on `.oui-paragraph` style. No need to add that in `.oui-tile` style.

Comment (JeremyDec) [reply packages/oui-tile/tile.less]: Done

**Comment by ovh-cds** [comment]
CDS Report yarn-build-ui-kit#1143.0 ✘  
* build
  * build ✘

**Comment by ovh-cds** [comment]
CDS Report yarn-build#699.0 ✘  
* build &amp; test
  * build ✘  
  * test ✔

**Pull request merged**